tests/output/ConfWriterTest.py

```diff
@@ -72,3 +80,31 @@ def test_write(self):
             lines = f.readlines()
 
         self.assertEqual(result_file, lines)
+
+    def test_write_with_dir(self):
```

Comment:
you basically need the uut_dir only in this test so I'd say remove it from setup/teardown and do it here